### DISCUSSION TITLE
Would it be possible for php-imagik to get HEIC support?

### DISCUSSION THREAD

**Discussion by fideloper** [oerdnj/deb.sury.org#1731] [Ideas]
Apple devices are trending towards using HEIC image format. 

As a result, support for HEIC in Imagick has been requested by clients more and more.

I was curious if it would be feasible to include HEIC support in php-imagick?

Here's a reference: https://eplt.medium.com/5-minutes-to-install-imagemagick-with-heic-support-on-ubuntu-18-04-digitalocean-fe2d09dcef1

It requires:

* libde265
* libheif
* building imagick with the `--with-heic=yes` flag, e.g. `./configure --with-heic=yes`

**Comment by oerdnj** [comment]
Unfortunately, I don't have **any** spare cycles to include `imagemagick` into my package set. It's not lightweight package and it would require a lot of time to properly backport and forwardport all the changes.
